backend/src/test/java/com/github/chuettenrauch/mixifyapi/unit/mixtape/controller/TrackControllerTest.java

```diff
@@ -0,0 +1,134 @@
+                        .with(oauth2Login())
+                )
+                .andExpect(status().isBadRequest())
+                .andExpect(content().json(expectedJson, true));
+    }
+
+    @Test
+    void delete_whenNotLoggedIn_thenReturnUnauthorized() throws Exception {
+        this.mvc.perform(delete("/api/mixtapes/123/tracks/123"))
+                .andExpect(status().isUnauthorized());
+    }
+
+    private static Stream<Arguments> provideInvalidTrackJson() {
+        return Stream.of(
+                arguments(named("name | missing", """
+                                    {
+                                        "artist": "some artist",
+                                        "imageUrl": "http://path/to/image",
+                                        "providerUri": "spotify:track:234"
+                                    }
+                                """),
+                        """
+                                {
+                                    "name": "must not be blank"
+                                }
+                                """
+                ),
+                arguments(named("artist | missing", """
+                                    {
+                                        "name": "some name",
+                                        "imageUrl": "http://path/to/image",
+                                        "providerUri": "spotify:track:234"
+                                    }
+                                """),
+                        """
+                                {
+                                    "artist": "must not be blank"
+                                }
+                                """
+                ),
+                arguments(named("imageUrl | missing", """
+                                    {
+                                        "name": "some name",
+                                        "artist": "some artist",
+                                        "providerUri": "spotify:track:234"
+                                    }
+                                """),
+                        """
+                                {
+                                    "imageUrl": "must be an absolute URL or URL to uploaded file"
+                                }
+                                """
+                ),
+                arguments(named("imageUrl | missing", """
+                                    {
+                                        "name": "some name",
+                                        "artist": "some artist",
+                                        "imageUrl": "http://path/to/image"
+                                    }
+                                """),
+                        """
+                                {
+                                    "providerUri": "not a valid provider uri"
+                                }
+                                """
+                )
+        );
+    }
+}
```

Comment:
Ich bin bisschen verwirrt, warum du zwei TrackControllerTest files erstellt hast? Die eine scheint nur unauthorized Sachen zu testen und die hier die Standard CRUD methoden? Warum nicht in einer?